indico/htdocs/sass/modules/_papers.scss

```diff
@@ -63,3 +63,93 @@
         width: 4rem;
     }
 }
+
+.display-contribution-list {
```

Comment:
`paper-contribution-list` or something like that? the current name sounds like something more likely to be used in the contributions module